api/v08/kogitoserverless_build_types.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v08
+
+import (
+	"github.com/RHsyseng/operator-utils/pkg/olm"
+	"github.com/ricardozanini/kogito-builder/api"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// KogitoServerlessBuildSpec defines the desired state of KogitoServerlessBuild
+type KogitoServerlessBuildSpec struct {
```

Comment:
please apply the kubebuilder annotations so the types are correctly added to the crd.
please add to all types.